internal/check/handler.go

```diff
@@ -70,7 +73,7 @@ type getCheckRequest struct {
 	MaxDepth int `json:"max-depth"`
 }
 
-// swagger:route GET /check read getCheck
+// swagger:route GET /check-sdk read getCheck
```

Comment:
> Alternatively it could be /check/<method> or /allowed/<method> where <method> could be for example openapi or envoy, traefik, ...

Makes sense :+1: 
The problem with query parameter is that no status should be default for SDK, but not default for others.

Reply:
Ah, that makes sense yeah. Another option is to use the user-agent, however I think that is not strict enough for security purposes.

Reply:
I think you had a good point there :wink: https://github.com/ory/keto/issues/851#issuecomment-1065045335

Reply:
Agreed :)

Reply:
Changed to `check/openapi`.